nexus/completion.go

```diff
@@ -115,11 +140,14 @@ type CompletionHandlerOptions struct {
 	// A stuctured logging handler.
 	// Defaults to slog.Default().
 	Logger *slog.Logger
+	// A [Serializer] to customize handler serialization behavior.
+	// By default the handler handles, JSONables, byte slices, and nil.
+	Serializer Serializer
```

Comment:
I'd be interested in knowing what the default serializer does when the handler returns something not in this list.

Reply:
It's very much like the default payload converter in temporal SDKs.
It will fail if it can't serialize to JSON.